r/R/Field.R

```diff
@@ -64,14 +64,17 @@
 #' @param metadata currently ignored
 #'
 #' @examples
-#' field("x", int32())
+#'
+#' if(arrow_available()){
```

Comment:
For Spark, TF, and reticulate we ended up doing a bunch of `\dontrun` entries. That might be more elegant than implying to users that they need to constantly check `arrow_available()` when actually using the package. Note that it's a flag to not have any runnable examples though so we'd have code for anything that could be run (+ code to demonstrate the use of the availability checking mechanism).

Reply:
Thanks.